src/sdks/core/sdk.config.json

```diff
@@ -0,0 +1,138 @@
+{
+    "info": {
+        "title": "Firebolt Core SDK"
+    },
+    "methods": [
```

Comment:
wonder if methods is the right name still?

Reply:
yeah, i was trying to keep it "inspired" by OpenRPC, since lots of the stuff is copied from here directly into the output OpenRPC doc.

It's basically a list of method "queries" so I'd be down to call it `method-queries`

Reply:
I say keep it as `methods` unless you want to discuss it further.